torch/distributed/distributed_c10d.py

```diff
@@ -44,6 +44,17 @@
 except ImportError:
     _GLOO_AVAILABLE = False
 
+# Some reduce ops are not supported by complex numbers.
```

Comment:
the way this comment is written reads like we allow calling them.

Reply:
@gchanan quick fix here: https://github.com/pytorch/pytorch/pull/46599